### ISSUE TITLE
Allow specifying events as 'must run after [other event]'

### ISSUE THREAD

**Issue by lukegb** [emfcamp/Website#1354]
<jonty> I've just remembered that I wanted to add a "this event must be after this other event" thing, for workshops that happen after talks as that almost went wrong more than once last year. Really easy to implement scheduling wise, but the UI is going to be ugly.

**Comment by crablab** [comment]
I don't think we have many of these this year, offhand. At least, I'm not aware of any so far.
